src/main/java/com/example/bookstoreweb/config/SecurityConfig.java

```diff
@@ -36,7 +41,16 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 .authenticated()
                 )
                 .httpBasic(Customizer.withDefaults())
+                .addFilterBefore(jwtAuthenticationFilter,
```

Comment:
JWT requires stateless session management (syntax may slightly vary depending on springboot version)
```suggestion
                .sessionManagement(sessionManagement ->
                        sessionManagement.sessionCreationPolicy(STATELESS))
                .addFilterBefore(jwtAuthenticationFilter,
```